trio/_tools/gen_exports.py

```diff
@@ -24,10 +24,19 @@
 HEADER = """# ***********************************************************
 # ******* WARNING: AUTOGENERATED! ALL EDITS WILL BE LOST ******
 # *************************************************************
-# isort: skip
+# isort: skip_file
```

Comment:
I really think we could put this prelude through `isort` at least.

Reply:
Perhaps what we should do is just make `gen_exports` run `black` + `isort` on the files it produces.